models/JournalEntry.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../db');
+
+class JournalEntry {
+  static addJournalEntry(userId, title, entryBody, isPrivate) {
```

Comment:
For all static methods since they're invoked with class it'll be just as readable and less redundant to take the name of the class away from method names. E.g. `JournalEntry.addJournalEntry()` vs. `JournalEntry.addEntry()` or `JournalEntry.add()`